src/pilot/assign_role.py

```diff
@@ -1288,14 +1276,21 @@ def ensure_node_is_powered_off(drac_client):
 def change_physical_disk_state_wait(
         node_uuid, ironic_client, drac_client, mode,
         controllers_to_physical_disk_ids=None):
-    reboot_required, job_ids = change_physical_disk_state(
-        drac_client, mode, controllers_to_physical_disk_ids)
+
+    physical_disk_status = drac_client.change_physical_disk_state(
```

Comment:
There is a PhysicalDisk.status attribute, so the above variable name is a bit confusing since at first glance, I thought it had something to do with that attribute.  Could we change the variable name to something else, like "change_state_result"?